python/lib/dcoscli/dcoscli/package/main.py

```diff
@@ -317,6 +317,7 @@ def _install(package_name, package_version, options_path, cli,
     :type yes: bool
     :returns: process status
     :rtype: int
+    :manager_id: str defines the custom manager to forward this operation to
```

Comment:
(applies to other parts of the file too). All the pydocs for method params should come before `:rtype`